### DISCUSSION TITLE
How to echo the field name using DSL?

### DISCUSSION THREAD

**Discussion by aborruso** [johnkerl/miller#1064] [Q&A] [answered]
Hi,
I have this input CSV:

```
id,name
1,Andrea
2,Marco
3,Luigia
```

How to echo a field name, after cutting it, via DSL?

Currently I do it running this `mlr --csv cut -f name input_01.csv | head -n 1`, but I'm sure there is a super easy and obvious way to do it using some DSL code and put.

Thank you and forgive the stupid question

**Comment by johnkerl** [comment, marked as answer]
@aborruso how about

```
mlr --csv --from input_01.csv head -n 1 then put -q 'print(joinv(get_keys($*), ","))'
```

?

**Reply by aborruso** [reply]
thank you
